README.md

```diff
@@ -3,64 +3,91 @@
+computer application, people call it a "**web application**."
 
-If all goes as it should, the entire process only takes a second or two. But
-there are a few other steps going on behind the scenes to make that heart red.
-Let's take a closer look at those steps.
+<figure>
+  <img src="https://curriculum-content.s3.amazonaws.com/fewpjs/fewpjs-fewp-example/legolas.png" alt="Twitter empty heart">
+  <figcaption>Web programmers see their favorite sites...differently</figcaption>
+</figure>
+
+## Identify Reference Example
+
+While this might be your first introduction to looking at some of your favorite
+sites with "web programmer" eyes, you've probably experienced plenty of web
+interactions that someone else programmed for you. For the rest of this
+material, we're going to pick _one, tiny_ interaction as our shared example.
+
+> **Web Programming Example**: "Favoriting" a social media post.
+
+Regardless of the social media site (Instagram, Pinterest, Facebook, LinkedIn,
+Twitter), the interaction went something like this:
 
-### Identify the "Three Pillars of Web Programming"
 
-We can break down web programming into three essential "pillars":
+1. The site renders some HTML content that is styled using CSS
+2. You see the content and decide to show your approval of it
+3. You _click_ some visual element meant to show approval (heart,
+   thumbs-up, +1, etc.) picture e.g. <img src="https://curriculum-content.s3.amazonaws.com/fewpjs/fewpjs-fewp-example/empty.png" alt="Twitter empty heart">
+4. The visual element _changes_ (animates, goes from empty to full, jiggles, etc) e.g.  <img src="https://curriculum-content.s3.amazonaws.com/fewpjs/fewpjs-fewp-example/full.png" alt="Twitter full heart">
+5. Behind the scenes, the application _tells the provider_ that such and such
+   post has gained one more approval (so that they can notify the creator or
+   re-share your approved message to your network)
 
-  - Manipulating the DOM
-  - Recognizing JS events
-  - Communication with the server
+If all goes as it should, the entire process only takes a second or two. But
+in this moment we did _all_ the work of front end web programming. As Blake
+might have [said][Blake]:
+
+> To see a World in a Grain of Sand  
+> And a Web Programming in a single Like
+
+### Identify the "Three Pillars of Web Programming"
 
-Using our "favoriting" example, here's how those pillars translate to our steps:
+We can break down web programming into three essential "pillars": In the
+previous list, we _italicized_ steps 3-5. Each of those demonstrated one of the
+"pillars" we must learn in order to be web programmers.
 
-1. Your click action on the empty heart triggers a JavaScript event.
-2. JavaScript sends a notification to the server.
-3. After a successful post to the server, the event updates the DOM with a full heart.
+  - Step 3 showed "**Recognizing JS events**:" Your click action on the empty heart
```

Comment:
@jenmyers if these are the definitions of the pillars we're going to use, let's put them in the bible